.github/workflows/publish_pytorch_dev_docker.yml

```diff
@@ -71,4 +72,89 @@ jobs:
             AMDGPU_TARGETS=${{ matrix.targets.amdgpu_target }}
           push: true
           tags: ${{ steps.meta.outputs.tags }}
-          labels: ${{ steps.meta.outputs.labels }}
+          labels: |
+            org.opencontainers.image.title=PyTorch ROCm Dev Image
+            org.opencontainers.image.description=Nightly ROCm PyTorch Dev Docker for target ${{ matrix.targets.amdgpu_target }}
+            org.opencontainers.image.version=${{ env.VERSION }}
+            org.opencontainers.image.created=${{ steps.meta.outputs.created }}
+            org.opencontainers.image.revision=${{ github.sha }}
+
+      - name: Export digest
+        id: export
+        run: |
+          digest="${{ steps.build_push.outputs.digest }}"
+          case "${{ matrix.targets.amdgpu_target }}" in
+            gfx942) echo "gfx942=$digest" >> $GITHUB_OUTPUT ;;
```

Comment:
Same as above. You probably do not want to limit this to gfx942 even though you want to limit running `verify-image` to gfx942.